api-audit/src/main/java/com/capitalone/dashboard/evaluator/CodeQualityEvaluator.java

```diff
@@ -33,6 +36,11 @@
 import java.util.Optional;
 import java.util.Set;
 import java.util.stream.Collectors;
+import java.util.Iterator;
+
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
+import org.json.simple.parser.*;
```

Comment:
Please import the classes you need and not a generic import

Reply:
done with this change